helpers.py

```diff
@@ -18,36 +26,56 @@ def login_nodeconductor(driver, username, password):
     time.sleep(2)
     # Type username
     username_field = driver.find_element_by_css_selector('[ng-model="auth.user.username"]')
-    username_field.send_keys('Julia')
+    username_field.send_keys(username)
     # Type password
     password_field = driver.find_element_by_css_selector('[ng-model="auth.user.password"]')
-    password_field.send_keys('123')
+    password_field.send_keys(password)
     # Click to login
     login_field = driver.find_element_by_class_name('button-login')
     login_field.click()
     # Necessary time for reloading page
     time.sleep(5)
 
-def create_new_project(driver, project_name, project_description):
+def choose_definite_organization(driver, nec_organization):
+    organization_field = driver.find_element_by_css_selector('ul.nav-list span.customer-name')
+    organization_field.click()
+    organization = driver.find_element_by_link_text(nec_organization)
+    organization.click()
+    time.sleep(5)
+    dashboard_field = driver.find_element_by_link_text('Dashboard')
+    dashboard_field.click()
+    time.sleep(5)
```

Comment:
`time.sleep(5)` should not be called in the end of function, because it delays function execution time. We cannot tell for sure that we will need delay after function execution.